src/main/webapp/components/ErrorHandler.js

```diff
@@ -16,32 +17,44 @@ export default class ErrorHandler extends Component {
     super(props);
     this.state = {
       hasError: false,
+      unauthorized: false
     };
 
     this.props.history.listen((location, action) => {
       if (this.state.hasError) {
         this.setState({
           hasError: false,
+          unauthorized: false
         });
       }
     });
   }
  
   componentDidCatch(error, info) {
     console.error(error);
+    let unauthorized = false;
+    if(!isEmpty(error.message) && error.message.includes("401")) {
```

Comment:
Could be simpler to use [lodash get](https://lodash.com/docs/4.17.15#get). Given that `Utils.isEmpty` is essentially a rewrite of a [`lodash function`](https://lodash.com/docs/4.17.15#isEmpty), we should probably deprecate that and use lodash instead - no point in keeping unnecessary code around.

Reply:
you are right, I'll change in my #346 PR and here. Thanks!